test/core/end2end/end2end_tests.cc

```diff
@@ -64,6 +68,20 @@ Slice RandomBinarySlice(size_t length) {
 void CoreEnd2endTest::SetUp() {
   CoreConfiguration::Reset();
   initialized_ = false;
+#ifdef GPR_WINDOWS
```

Comment:
Let's move this out to a utility function (maybe in `test_config.h` as `grpc_prewarm_os_for_tests()`?) -- it's not unlikely that some other test suite will eventually need this.

Reply:
done, thanks Craig!